micrometer/src/test/java/org/jboss/eap/qe/micrometer/multiple/MultipleWarTestCase.java

```diff
@@ -20,24 +20,27 @@
 import org.jboss.eap.qe.micrometer.container.PrometheusMetric;
 import org.jboss.eap.qe.micrometer.util.MicrometerServerSetup;
 import org.jboss.eap.qe.microprofile.tooling.server.configuration.deployment.ConfigurationUtil;
+import org.jboss.eap.qe.ts.common.docker.junit.DockerRequiredTests;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Assert;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
 @RunWith(Arquillian.class)
 @ServerSetup(MicrometerServerSetup.class)
+@Category(DockerRequiredTests.class)
 public class MultipleWarTestCase extends BaseMultipleTestCase {
-    @Deployment(name = SERVICE_ONE, order = 1)
+    @Deployment(name = SERVICE_ONE, order = 1, testable = false)
```

Comment:
Why the client mode?

Reply:
This is to avoid some WARNs in the server logs. Got inspiration from here: https://github.com/wildfly/wildfly/pull/17898#discussion_r1671182878

Reply:
Nice tip, thanks!